### ISSUE TITLE
Adding Customer Group Pricing Rules

### ISSUE THREAD

**Issue by buddylindsey** [SummitESP/fishbowl-api#2]
My thought is that we add another constant which is, maybe, `CUSTOMER_GROUP_PRICING_RULES_SQL`. We use the query below and run it through the same code as to add to `pricing_rules` dict.

That would lend us to get all the pricing rules for all customers not only individually added customers, but for customers that are used in customer groups as well. What do you think?

You might run the query to see the results and what you think.

The last 3 fields are there strictly for visual confirmation of what data it is pulling in.

``` SQL
select p.id, p.isactive, product.num, p.patypeid, p.papercent, p.pabaseamounttypeid, p.paamount, p.customerincltypeid, p.customerinclid, c.id as customerid, ag.name as accountgroupname, c.name as customername
from pricingrule p
inner join product on p.productinclid = product.id
inner join accountgroup ag on p.customerinclid = ag.id
inner join accountgrouprelation agr on agr.groupid = ag.id
inner join customer c on agr.accountid = c.accountid
where p.productincltypeid = 2;
```

**Comment by SmileyChris** [comment]
Sounds good, do you want to go ahead and do this?

**Comment by buddylindsey** [comment]
I'll do it and open up a PR for you to verify and practical test.

**Comment by SmileyChris** [comment]
Merged in #3
